content/api/commands/mount.md

```diff
@@ -179,13 +184,6 @@ include the `cypress.d.ts` file in all your `tsconfig.json` files like so:
 "include": ["./src", "cypress.d.ts"]
 ```
 
-## Additional Mount Commands
```

Comment:
Is removing this and normalizing the custom command names just for simplicity?

Reply:
Feedback from Jess was that we shouldn't have "additional" mount commands, that a dev should only have one mount command that takes care of all the setup needed. So I removed the title and updated the examples to all by `cy.mount` instead of `cy.mountXXX`.

Reply:
👍